types/node/child_process.d.ts

```diff
@@ -110,7 +124,13 @@ declare module "child_process" {
         windowsVerbatimArguments?: boolean;
     }
 
+    interface SpawnOptionsWithoutStdio extends SpawnOptions {
+        stdio?: 'pipe' | Array<undefined | 'pipe'>;
```

Comment:
I think `null` is also allowed as member in the array.

Reply:
Is `null` included in the documentation though? I ask this because as long as something is not in the documentation, Node.js developers are free to change the behavior without concern of breaking changes.

Reply:
see https://nodejs.org/dist/latest-v11.x/docs/api/child_process.html#child_process_options_stdio
`null, undefined - Use default value. For stdio fds 0, 1, and 2 (in other words, stdin, stdout, and stderr) a pipe is created. For fd 3 and up, the default is 'ignore'.` 

like in a lot cases `null` and `undefined` are interchangeable here.

Reply:
OK. I'll fix this. But I have another question: Since `options.stdio` itself can also be `null`, should I also modify `SpawnOptions.stdio`'s type from `StdioOptions` to `StdioOptions | null`?

Reply:
I don't think it's needed to allow `options.stdio === null` as it's not documented.
Actual implementation in node does `options.stdio || 'pipe'` so actually everything falsy like `false`, `0` or `''` ends up in `pipe` but I don't think we should allow this in the type definitions except if there is really a good reason to do this.